dependent/plastic-and-reconstructive-surgery-global-open.csl

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/npg -->
+  <info>
+    <title>Plastic and Reconstructive Surgery. Global open.</title>
+    <id>http://www.zotero.org/styles/plastic-and-reconstructive-surgery-global-open</id>
+    <link href="http://www.zotero.org/styles/plastic-and-reconstructive-surgery-global-open" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="independent-parent"/>
+    <link href="http://edmgr.ovid.com/prs/accounts/ifauth.htm" rel="documentation"/>
```

Comment:
Can you change this to http://journals.lww.com/prsgo/Pages/informationforauthors.aspx?

(I don't like linking to pages like http://edmgr.ovid.com/prs/accounts/prs_go_ifauth.htm, which don't offer a clear link back to the journal homepage, so I rather go one level back to http://journals.lww.com/prsgo/Pages/informationforauthors.aspx)